flux_inference/flux_model.py

```diff
@@ -0,0 +1,104 @@
+import json
+from pathlib import Path
+import gc
+from PIL import Image
+
+import torch
+from safetensors.torch import load_file 
+from optimum.quanto import freeze, qfloat8, quantize, requantize
```

Comment:
NOTE: Lora loading has been fixed at  optimum quanto commit `f9b71f4`, but later commits break the code again. There has been no new release which includes the required changes yet, so the module must be built from github.